Packs/FeedGitHub/Integrations/FeedGitHub/FeedGitHub.py

```diff
@@ -120,10 +121,11 @@ def get_content_files_from_repo(client: Client, relevant_files: list[str], param
         list: A list of file contents fetched via HTTP requests.
     """
     global RAW_RESPONSE
-    resp_type = "json" if params.get("feedType") == "STIX" else "text"
     extensions_to_fetch = argToList(params.get("extensions_to_fetch") or [])
     relevant_files = [file for file in relevant_files if any(file.endswith(ext) for ext in extensions_to_fetch)]
-    raw_data_files = [{file: client._http_request("GET", full_url=file, resp_type=resp_type)} for file in relevant_files]
+    raw_data_files = [
+        {file: base64.b64decode(client._http_request("GET", url_suffix=f"/contents/{file}")["content"]).decode("utf-8")} for file
+        in relevant_files]
```

Comment:
Did you check the removal of the `resp_type` for all `feedType` options?
The `content` key always appears? Also for empty files and for all file types?